### DISCUSSION TITLE
Magicento for PHPStorm

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2497] [Questions & Answers] [answered]
PHP is not my favorite programming language but I can deal with it. I am using PHPStorm for some time and for debugging I have a pretty good configuration. Recently someone told me about Magicento for Magento 2 and I found out there is version for Magento 1too. Does it worth to install it in PHPStorm?

PS - Don't forget to buy a PHPStorm license in the next period. The prices will rise up from October.

**Comment by fballiano** [comment, marked as answer]
1000000% worth! I bought it 3 or 4 times since the updates are downloadable only for 1year since the buying date.

**Reply by fballiano** [reply]
The M2 version is less valuable IMHO since adobe’s pfficial phpstorm plugin is already pretty complete

**Reply by addison74** [reply]
Good to know related to M2 version.
